.github/workflows/dotnet.yml

```diff
@@ -1,6 +1,3 @@
-# This workflow will build a .NET project
-# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
-
 name: .NET
 
 on:
```

Comment:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch